Makefile

```diff
@@ -72,3 +72,35 @@ docs:
 .PHONY: docs
 docker-prune:
 	./scripts/prune_all.sh
+
+.PHONY: check
+check:
+	SKIP_WASM_BUILD= cargo check --features all-frequency-features
+
+check-local:
+	SKIP_WASM_BUILD= cargo check --features  frequency-rococo-local
+
+check-rococo:
+	SKIP_WASM_BUILD= cargo check --features  frequency-rococo-testnet
+
+check-mainnet:
+	SKIP_WASM_BUILD= cargo check --features  frequency
+
+.PHONY: build
+build:
+	cargo build --locked --release --features all-frequency-features
+
+build-local:
+	cargo build --locked --release --features  frequency-rococo-local
+
+build-rococo:
+	cargo build --locked --release --features  frequency-rococo-testnet
+
+build-mainnet:
+	cargo build --locked --release --features  frequency
+
+build-rococo-release:
+	cargo build --locked --features  frequency-rococo-testnet --profile production
+
+build-mainnet-release:
+	cargo build --locked --features  frequency --profile production
```

Comment:
add some additional make commands, however cargo check/build will still generate full binary, so nothing will break for devs in their current way of working

Reply:
I wonder if it makes sense to add Make rules for the production target spec generation

Reply:
it is used in a github job to generate specs and binary, mainly used for our releases unless someone wants to check things out locally